packages/truffle/webpack.config.js

```diff
@@ -5,6 +5,14 @@ const webpack = require("webpack");
 const pkg = require("./package.json");
 const rootDir = path.join(__dirname, "../..");
 const outputDir = path.join(__dirname, "build");
+const truffleLibraryDirectory = path.join(
+  __dirname,
+  "../..",
+  "node_modules",
+  "@truffle/resolver",
+  "lib",
```

Comment:
I think @truffle/resolver's `yarn prepare` should put these in `packages/resolver/dist/` instead of having `truffle` grab them from `lib`.

(As it is, there's an inconsistency: @truffle/resolver's `package.json` includes `dist` in `files`, but not `lib`, so this only works in the monorepo, not that we care about that here...)